core/bin/block_reverter/src/main.rs

```diff
@@ -46,6 +48,10 @@ struct Cli {
     /// Path to yaml genesis config. If set, it will be used instead of env vars
     #[arg(long, global = true)]
     genesis_path: Option<PathBuf>,
+    /// Path to yaml config of the chain on top of gateway.
+    /// It may be `None` in case a chain is not settling on top of Gateway.
```

Comment:
Nit: For external users (not sure whether the block reverter has them), speaking in terms of Rust types is unnecessarily low-level. Something like "Should be skipped in case..." is more understandable.